rmc-docs/src/rmc-tutorial.md

```diff
@@ -0,0 +1,9 @@
+# RMC tutorial
+
+This tutorial will step you through a reasonable progression from simple to moderately complex tasks with RMC.
```

Comment:
Remove 'reasonable'. Maybe start with some motivation. Why do you want to invest time learning RMC?

Reply:
Did the first. I also added more context.